common/app/views/fragments/header.scala.html

```diff
@@ -5,132 +5,174 @@
-                                    popup-container has-popup js-profile-register
-                                    hide-until-mobile-landscape">
-                            <a href="@Configuration.id.membershipUrl?INTCMP=DOTCOM_HEADER_BECOMEMEMBER"
-                                data-link-name="Register link"
-                                class="brand-bar__item--action">
-                                    @fragments.inlineSvg("profile-add-36", "icon", List("rounded-icon", "control__icon-wrapper"))
-                                <span class="brand-bar__item__badge control__icon-wrapper control__icon-wrapper--text">free</span>
-                                <span class="control__info js-control-info control__info--supporting">become a member</span>
-                            </a>
-                        </div>
-                    }
-                    @if(IdentityProfileNavigationSwitch.isSwitchedOn) {
-                        <div class="brand-bar__item brand-bar__item--profile popup-container
-                                    has-popup js-profile-nav"
-                             data-component="identity-profile">
-                             <a href="@Configuration.id.url/signin?INTCMP=DOTCOM_HEADER_SIGNIN"
-                               data-link-name="User profile"
-                               data-toggle="popup--profile"
-                               data-toggle-signed-in="true"
-                               class="brand-bar__item--action popup__toggle"
-                               data-test-id="sign-in-link"
-                               aria-haspopup="true">
-                                @fragments.inlineSvg("profile-36", "icon", List("rounded-icon", "control__icon-wrapper"))
-                                @if(conf.switches.Switches.SaveForLaterSwitch.isSwitchedOn) {
-                                    @fragments.inlineSvg("bookmark", "icon", List("save-for-later__icon"))
-                                }
-                                <span class="js-profile-info control__info"
-                                      data-test-id="sign-in-name">sign in</span>
-                            </a>
-                            <div class="js-profile-popup">
-                                <ul class="popup popup--default popup__group popup--profile is-off" data-link-name="Sub Sections" data-test-id="popup-profile">
+@if(mvt.ABNewHeaderVariant.isParticipating) {
+    <header class="new-header" role="banner" data-link-name="global navigation: new header">
+        <div class="new-header__inner gs-container">
+            <a href="#" class="new-header__logo" tabindex="0">
+                <h1 class="u-h">The Guardian</h1>
+                @fragments.inlineSvg("guardian-logo-160", "logo")
+            </a>
+            <button class="new-header__edition-button">
+                <svg width="14" height="14" viewBox="0 0 14 14">
```

Comment:
Correct! It's easier to iterate if I don't have to worry about this sort of thing, but I'll leave a comment!